basicsynbio/functions/pdf_instructions.py

```diff
@@ -0,0 +1,409 @@
+            [
+                Paragraph(
+                    "Add 54 μl of magnetic beads into 96 well Falcon plate (one well per BASIC reaction) and add the 30 μl BASIC linker ligation from the PCR machine step, mix by pipetting 10 times. ",
+                    styles["BodyText"],
+                ),
+                Paragraph(
+                    "Wait 5min to allow DNA binding to magbeads. ",
+                    styles["BodyText"],
+                ),
+                Paragraph(
+                    "Place Falcon plate on magnetic stand and wait for rings to form and solution to clear.",
+                    styles["BodyText"],
+                ),
+                Paragraph(
+                    "Remove the solution with a 200 μl pipette tip from the centre of each well.",
+                    styles["BodyText"],
+                ),
+                Paragraph(
+                    "Add 190 μl 70'%' EtOH to each well and wait 30 s. ",
+                    styles["BodyText"],
+                ),
+                Paragraph(
+                    "Remove solution from each well (pipette set to 200 μl volume) ",
+                    styles["BodyText"],
+                ),
+                Paragraph(
+                    "Add 190 μl 70'%' EtOH to each well and wait 30 s. ",
+                    styles["BodyText"],
+                ),
+                Paragraph(
+                    "Remove solution from each well (pipette set to 200 μl volume)",
+                    styles["BodyText"],
+                ),
+                Paragraph("Leave the plate to dry for 1-2 min.", styles["BodyText"]),
+                Paragraph(
+                    "Remove Falcon plate from magnet and resuspend magbeads in 32 μl dH20. ",
+                    styles["BodyText"],
+                ),
+                Paragraph("Wait 1 min for DNA to elute. ", styles["BodyText"]),
+                Paragraph(
+                    "Place Falcon plate back on magnetic stand and allow ring to form and solution to clear. ",
+                    styles["BodyText"],
+                ),
+                Paragraph(
+                    "Pipette 30 μl of H20 with eluted DNA into fresh 1.5 ml Eppendorf tube for direct use in assembly or storage at -20°C for up to 1 month. ",
+                    styles["BodyText"],
+                ),
+            ]
+        )
+    )
+    # Assembly Reaction 3
+    elems.append(
+        Paragraph(
+            "Assembly reaction",
+            styles["Heading2"],
+        )
+    )
+    elems.append(
+        Paragraph(
+            "Below contains a table with each BASIC assembly within your build",
+            styles["BodyText"],
+        )
+    )
+    elems.append(Spacer(1, 0.4 * cm))
+    elems.append(
+        Table(ASSEMBLIES_DATA, colWidths=[5.2 * cm, 5.2 * cm, 5.2 * cm], style=style)
+    )
+    elems.append(
+        Paragraph(
+            "For each BASIC assembly (example here with 3 parts) combine parts with buffer in a PCR tube: ",
```

Comment:
"For each BASIC assembly, combine the required purified clip reactions in a final volume of 10 μl in 1x Assembly or NEB CutSmart buffer. Below gives an example for a 3-part assembly:"